src/pages/studyView/tabs/SummaryTab.tsx

```diff
@@ -343,6 +343,12 @@ export class StudySummaryTab extends React.Component<
                 break;
             }
             case ChartTypeEnum.SAMPLE_TREATMENTS_TABLE: {
+                props.description = {
+                    description:
```

Comment:
where is this used?

Reply:
I did a useage search- it was only used in `ChartHeader` to create the tooltip. My guess is that the extra fields are an artifact of this coming from an API endpoint for the survival charts.

Reply:
Ok. I see there is `description` is `ChartMeta`. Can we use that?

Reply:
I like that idea- sure.